docs/permissions.md

```diff
@@ -4,13 +4,9 @@ After the data controller has [approved your project](https://www.opensafely.org
 
 Additionally, the OpenSAFELY framework requires a user to be a member of the `opensafely` Github organisation in order to [run jobs in the secure backends](job-server.md).
 
-## Adding collaborators
 
-While your repository is private, you may need to add new collaborators who are not named in your original application. Any new collaborators have the same information governance obligations as the original study authors, and must therefore sign the same access agreements; please send any such requests to [team@opensafely.org](mailto:team@opensafely.org).
-
-When you add a new repository in the opensafely organisation, it may take up to 15 mintutes for it to be available to select at https://jobs.opensafely.org
-
-## Project Member Management
```

Comment:
I've removed this page as part of #897 and folded all the information into the jobs site page. Would you like to check that this fits with hat you had in mind?

Reply:
Fab, yes that's perfect. I'll not merge any changes to that file then to avoid conflicts.

The only thing I'd suggest is that you link to the https://docs.opensafely.org/publishing-repo/ page in the `Publishing outputs` section otherwise you risk repeating information.